scAR/test/test_activation_functions.py

```diff
@@ -0,0 +1,35 @@
+""" This tests the activation functions. """
+import decimal
+import unittest
+import numpy
+import torch
+from decimal import Decimal
+from scAR.main._activation_functions import mytanh, hnormalization, mySoftplus
+
+
+class ActivationFunctionsTest(unittest.TestCase):
+    """
+    Test activation_functions.py functions.
+    """
+
+    def test_mytanh(self):
+        """
+        Test mytanh().
+        """
+        self.assertEqual(mytanh(Decimal(1)).quantize(decimal.Decimal('.01'), rounding=decimal.ROUND_DOWN),
+                         Decimal(0.88).quantize(decimal.Decimal('.01'),
+                                                rounding=decimal.ROUND_DOWN))
+
+    def test_hnormalization(self):
```

Comment:
@CaibinSh, as you can notice, the data used to run these tests are probably not really relevant. If you have specific Tensor matrices we should be using to run the tests, please let me know.

Keep in mind that we cannot generate these matrices randomly as we need to be able to predict what the results of these functions will be (that's the main purpose of testing).

Reply:
@mr-nvs , thanks for reminding me. The current version is good. A tensor matrix is not necessarily needed once the output values are expected. The ML framework will ensure the input as tensor matrices.